### PR TITLE
Add additional host, machine and node selectors

### PR DESCRIPTION
**Pull request by jtomasek**
Ref https://github.com/kubevirt/web-ui/pull/301

### CODE REVIEW

File: src/selectors/node/selectors.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { get } from 'lodash';
+
+export const getNodeUnschedulable = node => get(node, 'spec.unschedulable', false);
```
Comment (suomiy) [inline src/selectors/node/selectors.js]: isNodeUnschedulable?

File: src/utils/status/host/hostStatus.js [hunk 2]
```diff
@@ -18,3 +23,5 @@ export const getHostStatus = host => {
 export const getSimpleHostStatus = host => getHostStatus(host).status;
 
 export const canHostAddMachine = host => [HOST_STATUS_READY].includes(getSimpleHostStatus(host));
+export const canHostStartMaintenance = hostNode => hostNode && !getNodeUnschedulable(hostNode);
+export const canHostStopMaintenance = hostNode => hostNode && getNodeUnschedulable(hostNode);
```
Comment (suomiy) [inline src/utils/status/host/hostStatus.js]: no need for `hostNode && ` in `canHostStopMaintenance` as getNodeUnschedulable returns false as default

File: src/selectors/machine/selectors.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+import { find, get } from 'lodash';
+
+export const getMachineNode = (nodes, machine) =>
```
Comment (suomiy) [inline src/selectors/machine/selectors.js]: can you please move this to `combined.js`? There is a convention for this name in other folders when the selector includes multiple entities.

File: src/selectors/host/selectors.js [hunk 4]
```diff
@@ -7,4 +7,6 @@ export const getHostRole = hostMachine =>
   get(hostMachine, ['metadata', 'labels', 'machine.openshift.io/cluster-api-machine-role']);
 
 export const getHostMachineName = host => get(host, 'spec.machineRef.name');
+export const getHostBmcAddress = host => get(host, 'spec.bmc.address');
 export const getHostErrorMessage = host => get(host, 'status.errorMessage');
+export const getHostPoweredOn = host => _.get(host, 'status.poweredOn');
```
Comment (suomiy) [inline src/selectors/host/selectors.js]: `isHostPoweredOn` ? maybe also good to indicate default as false?

File: src/selectors/node/selectors.js (hunk 1 above)
Comment (jtomasek) [reply src/selectors/node/selectors.js]: Done

File: src/utils/status/host/hostStatus.js (hunk 2 above)
Comment (jtomasek) [reply src/utils/status/host/hostStatus.js]: Done

File: src/selectors/machine/selectors.js (hunk 3 above)
Comment (jtomasek) [reply src/selectors/machine/selectors.js]: Done

File: src/selectors/host/selectors.js (hunk 4 above)
Comment (jtomasek) [reply src/selectors/host/selectors.js]: Done

File: src/utils/status/host/tests/hostStatus.test.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import { getHostStatus, getSimpleHostStatus } from '../hostStatus';
+import hostFixtures from '../fixtures/hostStatus.fixture';
+
+describe('getHostStatus()', () => {
```
Comment (suomiy) [inline src/utils/status/host/tests/hostStatus.test.js]: please remove this method or add `.expected` results to fixtures

**Pull request merged**